corehq/apps/es/client.py

```diff
@@ -131,6 +131,10 @@ def cluster_routing(self, *, enabled):
         value = "all" if enabled else "none"
         self._cluster_put_settings({"cluster.routing.allocation.enable": value})
 
+    def cluster_get_settings(self, is_flat=True):
```

Comment:
The option would be valuable when we have to get the settings as a `dict`. As of now we are not using it anywhere but in a scenario when we have to get all the settings for a particular attribute, it could be helpful. 
I can add tests for that.